src/components/Body/FilterItem.js

```diff
@@ -0,0 +1,10 @@
+function FilterItem(props) {
+    return (
+        <div onClick={(e) => props.filter(e.target.innerHTML)}>
```

Comment:
Ovo se ne radi - security hole. Podaci za filtriranje trebaju biti snimljeni u kontekstu - ono `ProductContext` gore iz komentara 

```suggestion
        <div onClick={(e) => props.filter(e.target.innerHTML)}>
```